library/packages/src/lib/y2packager/package.rb

```diff
@@ -43,6 +43,23 @@ def find(name)
         return nil if props.nil?
         props.map { |i| new(i["name"], i["source"], i["version"]) }
       end
+
+      # Find the highest version of requested package with given statuses
+      #
+      # @param name [String] name of searched package
+      # @param statuses [Array<Symbol>] allowed package statuses
+      #
+      # @return [Y2Packager::Package, nil] Highest found version of package; nil if not found
+      def last_version(name, statuses: [:available, :selected])
+        packages = find(name)
+
+        return nil unless packages
+
+        packages
+          .select { |i| [:available, :selected].include?(i.status) }
```

Comment:
you should use here that `statuses` and when you will do that force push, please also update commit message with new name of method.

Reply:
Oops! You know what people says: "Speed isn't good" :man_facepalming: I'll fix it in less than a second :running_man: 

:)